libs/auth-api-lib/migrations/20220823071133-rename-supportsdelegation-to-supportscustomdelegation.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    queryInterface.renameColumn(
+      'client',
+      'supports_delegation',
+      'supports_custom_delegation',
+    )
```

Comment:
This migration could cause a lot of pain during release. Can we create a new column instead with the same data as supports_delegation, and clean up the old column later?